apps/angular/src/app/features/anime/anime-details-page/anime-details-page.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="anime">
+	<ng-container *ngIf="anime$ | async as anime">
+		<h2 class="anime__title">{{ anime.titleEnglish | empty }}</h2>
+		<h2 class="anime__title">{{ anime.titleJapanese | empty }}</h2>
+		<div class="anime__details">
+			<img
+				class="anime__details-image"
+				[alt]="anime.titleEnglish"
+				[src]="anime.imageUrl"
+				(click)="openDialog(anime.imageUrl, anime.titleEnglish)"
+			/>
+			<div>
+				<p><strong>Season:</strong> {{ anime.season | readable }}</p>
+				<p><strong>Synopsis:</strong> {{ anime.synopsis }}</p>
+				<p><strong>Airing:</strong> {{ anime.airing | yesOrNo | empty }}</p>
+				<p><strong>Date start:</strong> {{ anime.aired.start | date | empty }}</p>
+				<p><strong>Date end:</strong> {{ anime.aired.end | date | empty }}</p>
+				<p><strong>Genres:</strong> {{ anime.genresData | readable | empty }}</p>
+				<p><strong>Studios:</strong> {{ anime.studiosData | readable | empty }}</p>
+				<p><strong>Rating:</strong> {{ anime.rating | readable | empty }}</p>
+				<p><strong>Source:</strong> {{ anime.source | readable | empty }}</p>
+				<p><strong>Type:</strong> {{ anime.type | readable | empty }}</p>
+				<p><strong>Status:</strong> {{ anime.status | readable | empty }}</p>
+				<p><strong>Created:</strong> {{ anime.created | date | empty }}</p>
+				<p><strong>Modified:</strong> {{ anime.modified | date | empty }}</p>
+			</div>
+		</div>
+		<ng-container *ngIf="saveVideoUrl$ | async as url">
+			<iframe
+				allowfullscreen
+				class="anime__video"
+				allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+				title="YouTube video player"
+				[src]="url"
+			></iframe>
+		</ng-container>
+	</ng-container>
+</div>
+<div *ngIf="isLoading$ | async" class="anime__spinner">
+	<camp-spinner />
+</div>
```

Comment:
Can we avoid redundant `div` and set structural directive with class directly to `camp-spinner`?
```suggestion
<camp-spinner *ngIf="isLoading$ | async" class="anime__spinner" />
```